concert/storage.py

```diff
@@ -221,11 +223,12 @@ class DirectoryWalker(Walker):
     specific filename template.
     """
 
-    def __init__(self, writer=TiffWriter, dsetname='frame_{:>06}.tif', bytes_per_file=0,
-                 root=None, log=None, log_name='experiment.log'):
+    def __init__(self, writer=TiffWriter, dsetname='frame_{:>06}.tif', start_index=0,
+                 bytes_per_file=0, root=None, log=None, log_name='experiment.log'):
         """
-        Use *writer* to write data to files with filenames with a template
-        from *dsetname*.
+        Use *writer* to write data to files with filenames with a template from *dsetname*.
+        *start_index* specifies the number in the first file name, e.g. for the default *dsetname*
+        and *start_index* 100, the first file name will be frame_000100.tif.
```

Comment:
@matze this is the best I can come up with in this heat.

Reply:
You mean in the harsh coldness of your AC'd office? ;-)

Reply:
Naaah, I leave the big guns off until the thermometer hits 32